meta-ros2-humble/recipes-bbappends/lgsvl-msgs/lgsvl-msgs_0.0.4-1.bbappend

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2020 LG Electronics, Inc.
```

Comment:
do we need these msgs? `LGSVL` is not supported anymore

Reply:
They are still in rosdistro: https://github.com/ros/rosdistro/blob/master/humble/distribution.yaml#L2184

The recipes will get generated.  These bbappends are just ensuring we get a clean build.